lib/wallaroo/core/autoscale/autoscale_phase.pony

```diff
@@ -740,11 +778,11 @@ class _WaitingForBoundariesToAckRegistering is _AutoscalePhase
 
   fun name(): String => "_WaitingForBoundariesToAckRegistering"
 
-  fun ref worker_join(conn: TCPConnection, worker: WorkerName,
-    worker_count: USize, local_topology: LocalTopology,
-    current_worker_count: USize)
+  fun ref try_join(local_topology: LocalTopologyInitializer,
```

Comment:
I don't think this should be implemented here, since we are in-progress for an autoscale event.